packages/ui-kit/src/components/Leaderboard/Leaderboard.stories.tsx

```diff
@@ -119,6 +135,7 @@ export const StaticStory: Story = {
 export const CustomStyleStory: Story = {
   name: 'Custom styles',
   tags: ['pattern'],
+  parameters: { imports: 'Leaderboard' },
```

Comment:
The cool thing is that the global story context settings can be overridden on the particular story level.